graylog2-server/src/main/java/org/graylog2/inputs/transports/HttpTransport.java

```diff
@@ -118,6 +134,10 @@ public static class Config extends AbstractTcpTransport.Config {
         @Override
         public ConfigurationRequest getRequestedConfiguration() {
             final ConfigurationRequest r = super.getRequestedConfiguration();
+            r.addField(new BooleanField(CK_ENABLE_BULK_RECEIVING,
+                    "Enable Bulk Receiving",
+                    false,
+                    "Enables bulk receiving of messages separated by newlines (\\n or \\r\\n)."));
```

Comment:
Very minor, but we should probably remove the period at the end here to maintain consistency with the surrounding checkbox fields.

Reply:
Good catch! Done.